mayastor/src/bdev/uring_util.rs

```diff
@@ -74,16 +74,17 @@ pub fn fs_type_supported(path: &str) -> bool {
     }
 }
 
-pub fn kernel_supports_io_uring() -> bool {
+pub fn kernel_supports_io_uring() -> Result<bool, String> {
```

Comment:
Since we have namespaces, we don't have to make that function name that long. How about just supported?  o we can write does:
```rust 
if uring_util::supported() { } 
```

Reply:
I've settled for kernel_support().